Marlin/src/module/stepper.cpp

```diff
@@ -3382,12 +3382,12 @@ void Stepper::endstop_triggered(const AxisEnum axis) {
       ) * double(0.5)
     #elif ENABLED(MARKFORGED_XY)
       axis == CORE_AXIS_1
-        ? count_position[CORE_AXIS_1] - count_position[CORE_AXIS_2]
+        ? count_position[CORE_AXIS_1] ENABLED(MARKFORGED_INVERSE, +, -) count_position[CORE_AXIS_2]
```

Comment:
That one gave me a compiler failure, can you check that?

Reply:
It should probably be `TERN(MARKFORGED_INVERSE, +, -)` instead of `ENABLED(MARKFORGED_INVERSE, +, -)`

Reply:
I've submitted a follow up PR:
- https://github.com/MarlinFirmware/Marlin/pull/26558